lib/ansible/modules/cloud/vmware/vmware_guest.py

```diff
@@ -40,6 +40,7 @@
     - "Module may require additional privileges as well, which may be required for gathering facts - e.g. ESXi configurations."
     - Tested on vSphere 5.5, 6.0, 6.5 and 6.7
     - Use SCSI disks instead of IDE when you want to expand online disks by specifying a SCSI controller
+    - Use SysPrep for Windows VM (depends on 'guest_id' parameter match 'win') with PyVmomi
```

Comment:
I am not sure the wording is 100% correct on it, "Use sysprep" makes it sound like you should sysprep any Windows Template beforehand, when actually its the opposite.  Maybe should be "Uses"?

Reply:
I agree with @cigamit. Could you please change wording a bit ? Thanks,.

Reply:
I totally agree, as English is not my native language wording and typos mistakes happen sometimes :/
Thanks for your vigilance!
I'm not a github specialist and this is obviously my first public commit/PR. I hope I did it good.
Regards

Reply:
Looks good to me.
shipit